test/e2e/network/ingress.go

```diff
@@ -603,11 +535,13 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 
 	Describe("GCE [Slow] [Feature:kubemci]", func() {
 		var gceController *framework.GCEIngressController
+		var ipName string
```

Comment:
Let each It block have its own ip address so that they can run in parallel?

Reply:
In fact, tested this locally and found this is working in parallel. `ipName` is being set up in BeforeEach, and each It block should have a pristine copy of the state.

Reply:
interesting. I thought they are sharing the same global variable, so all BeforeEach with overrite the same variable. AM surprised they each get their own copy